importing-data-5.html

```diff
@@ -86,58 +121,98 @@
 
 </li>
 
-<li><p class="first">Mouse-over the thumbnails of successfully imported images to see a preview.</p>
+<li><p class="first">Import progress will be shown in the <span class="bold">Import # 1</span> tab.</p>
+
+<p>Further data can be queued and imported using the <span class="bold">Select Data to Import</span> tab.</p>
 
-<p>Click on the <span class="bold">View</span> menu to view the image in the <span class="bold">Viewer</span> or <span class="bold">Data Manager</span>, or view the <span class="bold">Import Log</span> and the data <span class="bold">Checksum</span> verification.</p>
+<p>Multiple queues can be run simultaneously.</p>
 
-<p>or close the <span class="bold">Import</span> window to view the data in the <span class="bold">Data Manager</span>.</p>
+<p>A green check is shown in the import tab and in the top right corner of the main Data Manager window when the queue is successfully imported.</p>
+
+<p>Close the <span class="bold">Import</span> window and refresh the data tree to view the data in the <span class="bold">Data Manager</span>.</p>
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<img alt="../images/importingData5End.jpg" class="align-center" src="images/importingData5End.jpg" style="width: 82%;"  />
+<img alt="../images/gettingStarted5Complete.jpg" class="align-center" src="images/gettingStarted5Complete.jpg" style="width: 100%;" />
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+</li>
+
 <div class="note">
 <p class="note-title">Note</p>
 <p class="note-text">From Version 5, OMERO imports and stores image files unchanged, in their original format. Therefore there is no longer any need to archive files on import.</p>
-</div>
+</div><!-- End note -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Double-click on the thumbnail or select from the <span class="bold">View</span> menu to open the image in the full viewer.</p>
```

Comment:
Note `View` option is broken https://trac.openmicroscopy.org.uk/ome/ticket/12780
Might want to discuss this with @pwalczysko